src/client/views/play.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <!-- Our in-document javascript is injected here before the server sends this html out -->
+        <!-- Our in-document javascript is injected here before we send out the file required to monitor the loading animation -->
```

Comment:
The in document javascript **is** the file required to monitor the losing animation.

But it also is in charge of loading our audio.

Reply:
already reverted in my second commit

Reply:
Oh my bad 😖 I thought I was viewing all changes but it must have been a single commit!